backend/application.py

```diff
@@ -40,6 +42,83 @@ def build_dataset_structure(dataset_version, user=None, dataset=None):
     return r
 
 
+class GetSchema(handlers.UnsafeHandler):
+    """
+    Returns the schema.org, and bioschemas.org, annotation for a given
+    url.
+
+    This function behaves quite differently from the rest of the application as
+    the structured data testing tool had trouble catching the schema inject
+    when it went through AngularJS. The solution for now has been to make this
+    very general function that "re-parses" the current uri to figure out what
```

Comment:
It's not the `current uri`, maybe better to say that the `'url' request parameter is parsed`.